### ISSUE TITLE
Second navbar on CodeLand is bugged

### ISSUE THREAD

**Issue by thomasbnt** [forem/forem#9448]
**Describe the bug**
When you scroll on [CodeLand page](https://dev.to/codeland) and stop at the live view, the second navbar flashing.

**To Reproduce**

1. Go to https://dev.to/codeland
2. Scroll down to live view
4. See the second navbar flashing

**Expected behavior**

No flashing.

**Screenshots**
![dev to_codeland](https://user-images.githubusercontent.com/14293805/88197541-ec255980-cc42-11ea-8fa4-d8781fbd8811.png)
![dev to_codeland (1)](https://user-images.githubusercontent.com/14293805/88197548-ed568680-cc42-11ea-8470-be74c5c7a6ce.png)

*No gif for this issue, I don't have very clean extensions to do this kind of presentation.*

**Desktop:**

- OS: Manjaro
- Browser: Google Chrome
- Version: Version 83.0.4103.97 (Build officiel) (64 bits)

**Comment by Ridhwana** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @forem/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/forem/forem/blob/master/CONTRIBUTING.md).

**Comment by joshpuetz** [comment]
This only happens in Chrome as far as I can tell, and not in Safari or Firefox

**Comment by mirie** [comment]
Closing this as a won't fix as it's no longer relevant
